### PR TITLE
add performance standards

### PR DESCRIPTION
**Pull request by xavierlefevre**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/output-sql-alchemy-orm-query.mo.md">output-sql-alchemy-orm-query.mo.md</a>**: Title doesn't contain `[MO]` tag at the beginning
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/python-investigation-tools.mo.md">python-investigation-tools.mo.md</a>**: Title doesn't contain `[MO]` tag at the beginning
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/front/simulate-network-iphone.mo.md">simulate-network-iphone.mo.md</a>**: Title doesn't contain `[MO]` tag at the beginning
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/performance-decision-flow.s.md">performance-decision-flow.s.md</a>**: Title doesn't contain `[Standard]` tag at the beginning
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/performance-decision-flow.s.md">performance-decision-flow.s.md</a>**: Standard doesn't have an *Owner*, could it be you?
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/performance-decision-flow.s.md">performance-decision-flow.s.md</a>**: Standard doesn't have a *Checks* part, how could you call this an Standard?
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/performance-decision-flow.s.md">performance-decision-flow.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/performance-decision-flow.s.md">performance-decision-flow.s.md</a>**: You failed to mention a *Good Examples* 
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/cache-routes-using-varnish.mo.md">cache-routes-using-varnish.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/cache-routes-using-varnish.mo.md">cache-routes-using-varnish.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/cache-routes-using-varnish.mo.md">cache-routes-using-varnish.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/cache-routes-using-varnish.mo.md">cache-routes-using-varnish.mo.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/how-to-investigate-performance.mo.md">how-to-investigate-performance.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/how-to-investigate-performance.mo.md">how-to-investigate-performance.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/how-to-investigate-performance.mo.md">how-to-investigate-performance.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/how-to-investigate-performance.mo.md">how-to-investigate-performance.mo.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/minimize-number-sql-queries.mo.md">minimize-number-sql-queries.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/minimize-number-sql-queries.mo.md">minimize-number-sql-queries.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/minimize-number-sql-queries.mo.md">minimize-number-sql-queries.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/minimize-number-sql-queries.mo.md">minimize-number-sql-queries.mo.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/output-sql-alchemy-orm-query.mo.md">output-sql-alchemy-orm-query.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/output-sql-alchemy-orm-query.mo.md">output-sql-alchemy-orm-query.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/output-sql-alchemy-orm-query.mo.md">output-sql-alchemy-orm-query.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/output-sql-alchemy-orm-query.mo.md">output-sql-alchemy-orm-query.mo.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/python-investigation-tools.mo.md">python-investigation-tools.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/python-investigation-tools.mo.md">python-investigation-tools.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/python-investigation-tools.mo.md">python-investigation-tools.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/python-investigation-tools.mo.md">python-investigation-tools.mo.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/serve-images-as-static-files.mo.md">serve-images-as-static-files.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/serve-images-as-static-files.mo.md">serve-images-as-static-files.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/serve-images-as-static-files.mo.md">serve-images-as-static-files.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/backend/serve-images-as-static-files.mo.md">serve-images-as-static-files.mo.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/front/simulate-network-iphone.mo.md">simulate-network-iphone.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/front/simulate-network-iphone.mo.md">simulate-network-iphone.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/front/simulate-network-iphone.mo.md">simulate-network-iphone.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-performance-steps/performance/front/simulate-network-iphone.mo.md">simulate-network-iphone.mo.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  </tbody>
</table>


## Automatic reviewers
  
cc: @louiszawadzki, @tychota
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by tychota — CHANGES_REQUESTED** [review]
Point avec Xav pour en discuter !

File: performance/backend/backend-performance-tools.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## K6: Load-test your back-end with JavaScript written tests
+[K6 official website](https://k6.io/)
+Describe your routes test in JS and run them with simultaneous calls.
+
+## Postman: Document and test the global performance of your routes with an interface
+[Our BNP AM Postman configuration](https://github.com/theodo/bnp-postman)
+Test your routes from always the same software and network, to have a point of comparison.
+
+### iPhone: Simulate a network like LTE, 3G or Edge
```
Comment (Minishlink) [inline performance/backend/backend-performance-tools.md]: You can also install network link conditioner on your mac too (for simulators, it's easier to manipulate than going to the iPhone settings)

**Comment by Minishlink** [comment]
@xavierlefevre Tu peux parler de monitoring avec [Firebase performance monitoring](https://firebase.google.com/docs/perf-mon/) et Apollo Engine aussi :)

**Review by tychota — APPROVED** [review]

**Pull request merged**